index.html

```diff
@@ -4,7 +4,111 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Personal Website</title>
+
+    <link rel="stylesheet" href="assets/styles/main.css" />
+    <link rel="stylesheet" href="assets/styles/header.css" />
+    <link rel="stylesheet" href="assets/styles/footer.css" />
+    <script
+      src="https://kit.fontawesome.com/0069c24e64.js"
+      crossorigin="anonymous"
+    ></script>
```

Comment:
Do you need that? What does it?

Reply:
It gives me access to the fontawesome-kit  - because I will put icons to everything that's not fast enough to escape ;-)  - https://blog.fontawesome.com/introducing-font-awesome-kits/